solidity/contracts/bridge/Bridge.sol

```diff
@@ -96,19 +138,24 @@ contract Bridge is Ownable {
     ///         address.
     mapping(address => bool) public isVaultTrusted;
 
-    /// @notice Collection of all unswept deposits indexed by
+    /// @notice Collection of all revealed deposits indexed by
     ///         keccak256(fundingTxHash | fundingOutputIndex).
-    ///         The fundingTxHash is LE bytes32 and fundingOutputIndex an uint8.
+    ///         The fundingTxHash is LE bytes32 and fundingOutputIndex an uint32.
     ///         This mapping may contain valid and invalid deposits and the
     ///         wallet is responsible for validating them before attempting to
     ///         execute a sweep.
     ///
-    /// TODO: Explore the possibility of storing just a hash of DepositInfo.
-    mapping(uint256 => DepositInfo) public unswept;
+    mapping(uint256 => DepositInfo) public deposits;
+
+    /// @notice Maps the wallet public key hash (computed using HASH160 opcode)
+    ///         to the latest sweep computed as keccak256(txHash | txOutputValue).
+    mapping(bytes20 => bytes32) public sweeps;
```

Comment:
If I remember correctly we planned to rename it to `mainUtxo`.

Reply:
Indeed. See: https://github.com/keep-network/tbtc-v2/pull/106/commits/3556c9b054412b8fee028248574bd48e800e4cbd and also https://github.com/keep-network/tbtc-v2/pull/106/commits/ff3b59fa2e48eec63ea0f71fdce85d2812a45bd3